gef-ui/templates/partials/application-details.njk

```diff
@@ -47,8 +49,10 @@
   {% include "includes/application-details/about-exporter.njk" %}
   {% include "includes/application-details/automatic-cover.njk" %}
   {% include "includes/application-details/facilities.njk" %}
-  <form method="post" data-cy="form">
-  {% include "includes/application-details/submit.njk" %}
-  </form>
+  {%if not previewMode %}
+    <form method="post" data-cy="form">
+      {% include "includes/application-details/submit.njk" %}
```

Comment:
Is this submitting to TFM? As CHECKER cannot have EDIT rights (i.e. always in PreviewMode), how do they submit to TFM?

Reply:
that is a good point ... I know they cant edit the application but then what do we check when they want to submit ? Will dig into that one.

Reply:
Ive added another status to cover this scenario